src/pal/src/exception/signal.cpp

```diff
@@ -697,6 +697,42 @@ PAL_ERROR InjectActivationInternal(CorUnix::CPalThread* pThread)
 
 /*++
 Function :
+    signal_ignore_handler
+
+    Simple signal handler which does nothing
+
+Parameters :
+    POSIX signal handler parameter list ("man sigaction" for details)
+
+(no return value)
+--*/
+static void signal_ignore_handler(int code, siginfo_t *siginfo, void *context)
+{
+}
+
+
+void PAL_IgnoreProfileSignal(void)
+{
+#if !HAVE_MACH_EXCEPTIONS && defined(SIGRTMAX)
```

Comment:
@janvorli ```perf record -g -e cycles:ppp -e signal:*``` will record instructions samples as the cycle counter expires as well as signals.  If I want to inject a marker into a benchmark, I can use something like ```pkill -64 corerun```.  When I am profiling a server application, this allows me to sweep through various client configurations and add a marker when the stimulus changes.  When I analyze the profile, I can use `perf script | <filterscript> ` to analyze only the events between a specific set of marker.  This can be fed into other flows i.e.  `perf script | <filterscript> | <flamegraph flatten script> | <flamegraph svg script>`

For the filter script, I used an awk script like below to only print cycle events after the first `SIGRTMAX` event:
```awk 

/.*signal:signal_deliver: sig=.*/ { 
    trace = /=64/;
} 

/sched:/ || /signal:/ { 
    suppress=1 
} 

/cycles:ppp/ { 
    suppress=0 
} 

{ 
    if(trace && !suppress) print 
}
```